building/qubes-r3-building.md

```diff
@@ -34,7 +34,7 @@ Last, you may want to disable memory balancing on `dev26` but keep in mind the i
 Once you've built `dev26`, open a Terminal window to it and install the necessary dependencies (see [QubesBuilder](/doc/qubes-builder/) for more info):
 
 ~~~
-$ sudo dnf install git createrepo rpm-build make wget rpmdevtools dialog rpm-sign gnupg dpkg-dev debootstrap python2-sh
+$ sudo dnf install gpg git createrepo rpm-build make wget rpmdevtools python2-sh dialog rpm-sign dpkg-dev debootstrap PyYAML devscripts perl-Digest-MD5 perl-Digest-SHA
```

Comment:
The `gnupg` package name was correct, there is no `gpg` package.

Reply:
Both Fedora and Debian support use of gpg as proxy for gnupg